src/rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -505,7 +505,8 @@ docker pull {{ jobData.image }} \
 ## network consumption
 docker run --name $docker_name \
   --init \
-  --tty \
+  --attach STDOUT \
+  --attach STDERR \
```

Comment:
> While a client is connected to container’s stdio using docker attach, Docker uses a ~1MB memory buffer to maximize the throughput of the application. If this buffer is filled, the speed of the API connection will start to have an effect on the process output writing speed. This is similar to other applications like SSH. Because of this, it is not recommended to run performance critical applications that generate a lot of output in the foreground over a slow client connection. Instead, users should use the docker logs command to get access to the logs.

So it might degrade performance?

Reply:
Anyway, I think it's not critical

Reply:
Should be caused by this https://github.com/moby/moby/issues/25542.
Dose tty also have performance degradation?
If not, we may need to think about to de-multiplex STDOUT and STDERR with -tty by ourselves

Reply:
Or could we let the docker container directly write stdout and stderr to yarn log dir?

Reply:
I am not sure if it will degrade the performance. This description comes from "docker attach" command, here we use "docker run --attach". I'm not sure if the implementation are same for these two commands.

BTW, before this change, all container stdout and stderr will redirect to current stdout. This change just separate it. It not causes more message output to the current tty.

Reply:
could we let the docker container directly write stdout and stderr to yarn log dir? This is the safest way to be high perf

Reply:
Currently, we already redirected user command stdout to /pai/log/user.pai.log and stderr to /pai/log/user.pai.error. We can use these two files as stdout and stderr directly.